.github/workflows/helm-lint.yml

```diff
@@ -20,89 +20,89 @@
 name:  SDE Lint and Test Charts
 
 on:
-    push:
-      branches:
-       - main
+  ### Trigger the workflow on push on main branch and path charts/**
+  push:
+    branches:
+      - main
+    paths:
+      - 'charts/**'
+  ### Trigger the workflow on pull request on path charts/**
+  pull_request:
       paths:
-       - 'charts/**'
-
-    pull_request:
-      paths:
-       - charts/** 
+        - charts/**
+  ### Trigger the workflow manually 
        
-    workflow_dispatch:
+  workflow_dispatch:
       inputs:
         node_image:
           description: 'kindest/node image for k8s kind cluster'
           # k8s version from 3.2 release as default
-          default: 'kindest/node:v1.24.6'
+          default: 'kindest/node:v1.27.3'
           required: false
           type: string
         upgrade_from:
           description: 'chart version to upgrade from'
           # chart version from 3.2 release as default
-          default: '0.0.9'
+          default: '0.1.0'
           required: false
           type: string
 
 jobs:
-  lint-test:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
+    lint-test:
+        runs-on: ubuntu-latest
+        steps:
+        - name: Checkout
+          uses: actions/checkout@v3
+          with:
+            fetch-depth: 0
+
+        - name: Kubernetes KinD Cluster
+          uses: container-tools/kind-action@v1
+          with:
+            # upgrade version, default (v0.17.0) uses node image v1.21.1 and doesn't work with more recent node image versions
+            version: v0.20.0
+            # default value for event_name != workflow_dispatch
+            node_image: ${{ github.event.inputs.node_image || 'kindest/node:v1.27.3' }}
 
-      - name: Set up Helm
-        uses: azure/setup-helm@v3
-        with:
-          version: v3.10.3
```

Comment:
Can you please remove these 2 lines?
```         
with:
  version: v3.9.3
```
It's wrong in our TRG and will be removed there too in the near future